useful-scripts/importer-publisher-tests/index.ts

```diff
@@ -0,0 +1,365 @@
+import rimraf from 'rimraf';
+import FormData from 'form-data';
+import { ReleaseDataProps, SubjectArrayT } from 'types';
+import ZipDirectory from './utils/ZipDirectory';
+import Sleep from './utils/Sleep';
+
+// disable insecure warnings
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+dotenv.config();
+const { JWT_TOKEN, API_URL, TOPIC_ID } = process.env;
+
+const createPublication = async () => {
+  console.time('CreatePublication');
+  try {
+    const res = await axios({
+      method: 'post',
+      url: `${API_URL}/api/publications`,
+      data: {
+        title: `importer-testing-${v4()}`,
+        topicId: TOPIC_ID,
+        contact: {
+          teamName: 'testing',
+          teamEmail: 'johndoe@gmail.com',
+          contactName: 'John Doe',
+          contactTelNo: '123456789',
+        },
+      },
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${JWT_TOKEN}`,
+      },
+    });
+    console.timeEnd('CreatePublication');
+    console.log(`Created publication. Status code ${res.status}`);
+    const publicationId = res.data.id;
+    return publicationId;
+  } catch (e) {
+    console.error(e);
+  }
+};
+
+const createRelease = async (publicationId: string) => {
+  console.time('createRelease');
+  try {
+    const res = await axios({
+      method: 'POST',
+      url: `${API_URL}/api/publications/${publicationId}/releases`,
+      data: {
+        timePeriodCoverage: { value: 'AY' },
+        releaseName: 2222,
+        typeId: '1821abb8-68b0-431b-9770-0bea65d02ff0',
+        publicationId: 'c76c1bc4-0a01-4116-74cd-08d8eebbf0c1',
+        templateReleaseId: '',
+      },
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: `Bearer ${JWT_TOKEN}`,
+      },
+    });
+    console.timeEnd('createRelease');
+    const releaseId = res.data.id;
+    console.log(
+      `Release URL: ${API_URL}/publication/${publicationId}/release/${releaseId}/data`,
+    );
+    return releaseId;
+  } catch (e) {
+    console.error(e);
+  }
+};
+
+const extractZip = async () => {
+  const zippedFile = await globby('*.zip');
+  const cleanZipFile = zippedFile[0];
+  const archive = fs.existsSync(zippedFile[0]);
+  console.log('archive', archive);
+  if (archive) {
+    const zip = new StreamZip.async({ file: cleanZipFile });
+    const count = await zip.extract(null, './test-files');
+    console.log(`Extracted ${count} entries to test-files`);
+    await zip.close();
+  }
+
+  // rename csv file..
+  const csvGlob1 = await globby('./test-files/*.csv');
+  const csvGlob2 = csvGlob1[0];
+  const randomId = Math.floor(Math.random() * 1000000);
+
+  fs.rename(`${csvGlob2}`, `./test-files/testfile-${randomId}.csv`, e =>
+    e ? console.error(e) : '',
+  );
+
+  // rename the meta file...
+  const metaGlob = await globby('./test-files/*.meta.csv');
+  const metaGlob2 = metaGlob[0];
+
+  fs.rename(`${metaGlob2}`, `./test-files/testfile.meta.${randomId}.csv`, e =>
```

Comment:
I think it would be better to rename it `testfile-${randomId}.meta.csv` - to keep it consistent with how the data csv is named.